Remove unused parameter

helunwencser opened the pull request:
This PR removes the unused parameter.

larryliu0820 reviewed (commented):
Thanks! While we are at it, can we write at least one test case for ManagedTensor?

larryliu0820 reviewed (approved):
Thanks for adding the test.

helunwencser committed:
Remove unused parameter (#3858)

Summary:
This PR removes the unused parameter.


Test Plan:
Imported from GitHub, without a `Test Plan:` line.

```
sh test/run_oss_cpp_tests.sh ./extension/runner_util/test/
buck2 test fbcode//executorch/extension/runner_util/test:managed_tensor_test
```

Reviewed By: larryliu0820

Differential Revision: D58209941

Pulled By: helunwencser
